webui/src/lib/components/darkModeToggle.tsx

```diff
@@ -1,24 +1,32 @@
-import React, {FC, useContext} from "react";
-import {DarkModeSwitch} from "react-toggle-dark-mode";
-import {AppActionType, AppContext} from "../hooks/appContext";
+import React, { FC, useContext } from "react";
+import "@theme-toggles/react/css/Classic.css";
+import { Classic } from "@theme-toggles/react";
+import { AppActionType, AppContext } from "../hooks/appContext";
 
 const DarkModeToggle: FC = () => {
-    const {state, dispatch} = useContext(AppContext);
+    const { state, dispatch } = useContext(AppContext);
 
-    const toggleDarkMode = (isOn: boolean) => {
+    const toggleDarkMode = (toggled: boolean) => {
```

Comment:
Sorry, but at least for me, this is more confusing than the original.
Yeah and I know I wrote this, but still confusing.